conftest.py

```diff
@@ -79,6 +96,7 @@ def driver(fx_executable: str, opt_headless: bool, set_prefs: List[Tuple]):
     for opt, value in set_prefs:
         options.set_preference(opt, value)
     s = webdriver.Firefox(options=options)
+    s.implicitly_wait(opt_implicit_timeout)
```

Comment:
`s` doesn't make much sense here. Could you name this something or maybe just use `_`?

Reply:
I'll go with something else. I don't like to use _ if I'm going to return the object.